packages/ui/src/composites/wui-icon-box/index.ts

```diff
@@ -17,7 +18,7 @@ export class WuiIconBox extends LitElement {
   public static override styles = [resetStyles, elementStyles, styles]
 
   // -- State & Properties -------------------------------- //
-  @property() public size: Exclude<SizeType, 'inherit' | 'xxs'> = 'md'
+  @property() public size: Exclude<SizeType, 'xxs'> = 'md'
```

Comment:
I thought we needed `inherit` somewhere. Are you sure we can do this without breaking UI for another component?